ethereum-contracts/README.md

```diff
@@ -1,72 +1,62 @@
+3. Once the Oracle module has verified that the validators' aggregated signing power is greater than the specified threshold, it mints the appropriate amount of tokens and forwards them to the intended recipient.
 
-![alt text](./img/settings.png "Setting ")
+The Relayer service and Oracle module are under development here: https://github.com/swishlabsco/cosmos-ethereum-bridge.
+ 
+## Installation
+Install Truffle: `$ npm install -g truffle`  
+Install dependencies: `$ npm install`  
 
-### Running tests of specific contracts
 
-    `$ truffle test test/<test_name.js>`
+Note: This project currently uses solc@0.5.0, make sure that this version of the Solidity compiler is being used to compile the contracts and does not conflict with other versions that may be installed on your machine.
 
-## Debugging
+## Testing
+Run commands from the appropriate directory: `$ cd ethereum-contracts`  
+Start the truffle environment: `$ truffle develop`  
+In another tab, run tests: `$ truffle test`  
+Run individual tests: `$ truffle test test/<test_name.js>`  
 
-Once you run the contracts you will get the `TX HASH` number of each function call. If not, you can take the `tx` field from the response of the transaction if you print on the respective test file:
+Expected output of the test suite:
+![peggytestsuite](https://user-images.githubusercontent.com/15370712/58388940-34fb6700-7fda-11e9-9aef-6ae7b2442a55.png)
 
-  ```
-  { tx: '0x2e352263105ff582f3c8e5368e3029819a0862f2e3795279c0726b8b8e6c64f7',
-  receipt:
-   { transactionHash: '0x2e352263105ff582f3c8e5368e3029819a0862f2e3795279c0726b8b8e6c64f7',
-     transactionIndex: 0,
-     blockHash: '0x0ae0afbd7d711871e9dde6740ea613bdb22cd9fc4f4c68bac46c22a8bf7e0b5e',
-     blockNumber: 529,
-     gasUsed: 75600,
-     cumulativeGasUsed: 75600,
-     contractAddress: null,
-     logs: [ [Object] ],
-     status: '0x01',
-     logsBloom: '0x00000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000040002000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000200000000000000000000' },
-  logs:
-   [ { logIndex: 0,
-       transactionIndex: 0,
-       transactionHash: '0x2e352263105ff582f3c8e5368e3029819a0862f2e3795279c0726b8b8e6c64f7',
-       blockHash: '0x0ae0afbd7d711871e9dde6740ea613bdb22cd9fc4f4c68bac46c22a8bf7e0b5e',
-       blockNumber: 529,
-       address: '0x1b5163d8309bc23c225ba666507efd036dc1c677',
-       type: 'someType',
-       event: 'SomeEvent',
-       args: [Object] } ] }
-  ```
+## Security, Privacy, Risks
```

Comment:
this section should be more notorious. Maybe consider moving it to the top

Reply:
Fixed in next push https://github.com/musnit/peggy/pull/3/commits/d9f9786b35ec0cf98ebf2cb68cd260b059fe9757. The exact section itself has not been moved but a general disclaimer with the same intent has been placed at the top of the main README. Does this satisfy the core intent of this proposed change?